index.d.ts

```diff
@@ -1,4 +1,5 @@
 export type Category = 'O' | 'A' | 'B' | 'C' | 'D' | 'CD';
+export type BedType = 'ROB' | 'RVT' | 'CVK' | 'DVC' | 'NO';
```

Comment:
Personally I would prefer an enum for this

Reply:
I would agree, but this is a JS implementation, so not much use 🤷